src/com/sbezboro/standardplugin/listeners/PlayerPortalListener.java

```diff
@@ -52,8 +52,9 @@ public void onPlayerPortal(PlayerPortalEvent event) {
 				if (newEnd != null) {
 					event.setTo(new Location(newEnd, 100, 50, 0));
 				}
-				
-				plugin.getLogger().info(player.getDisplayName(false) + " going to the end.");
+				if (!(player.isPvpProtected())) {
```

Comment:
The line below is just logging to console, no need to restrict

Reply:
Added this so it doesn't log to console when PvP protected players try to enter end, the idea behind blocking pvp prot players from entering end is alt abusing during end resets with pvp timer

Reply:
Why remove the console log? Only I ever see it, just gives me an idea of the flow player in and out of the end

Reply:
oh, I am stupid.... I did if they are not pvp protected my bad, will change